snuba/query/parser/__init__.py

```diff
@@ -81,7 +81,9 @@ def build_selected_expressions(
     ) -> List[SelectedExpression]:
         output = []
         for raw_expression in raw_expressions:
-            exp = parse_expression(tuplify(raw_expression))
+            exp = parse_expression(
+                tuplify(raw_expression), dataset.get_abstract_columnset()
+            )
```

Comment:
Do we have expressions like `equals(stacktraces...., 'something')`  in the select clause or did you add the abstract columnset everywhere an expression is parsed since the parser cannot easily tell a function in the context of a condition from a function in a select/group by/ ... ?

Reply:
The latter. `parse_expression` is used in a lot of places and I needed to pass the column context through to `parse_function_to_expr`.